js/input_validator.js

```diff
@@ -0,0 +1,20 @@
+function isValidEmail (emailString) {
+//  https://www.ietf.org/rfc/rfc5322.txt
```

Comment:
This source does not back up the claimed regex (please provide page number if I am wrong). Please provide the actual source. https://emailregex.com/ ?